api/swagger.yml

```
@@ -4294,9 +4294,9 @@ paths:
       operationId: createPresignMultipartUpload
       summary: Initiate a multipart upload
       description: |
+        This operation is enabled only for the S3 blockstore.
```

I would rather not provide versioning info in the API.  For instance, say I have an _old_ client SDK, version 1.28.  And say you add support for GCS multipart in version 1.91.  Then the docs are simply incorrect.

I would prefer
```suggestion
        This operation is supported by the S3 blockstore, but might not be supported by all blockstores.  Refer to the lakeFS server documentation.
```

---

Documentation lives in docs/, and you may certainly change it :-)

You may be referring to the fact that this is an experimental feature, and as such is probably _not_ documented.  In that case, let's not commit to S3 support.  Ask @treeverse/product if they would like to document which backends are supported for now.